### PR TITLE
Meshack

### PR DESCRIPTION
**Pull request by Cheberi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,35 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+   <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (TenTraicion) [inline index.html:118]:
```suggestion
   <!-- ________ *Meshack Cheberi* Contributor card START ________  -->
```

File: index.html [hunk 2]
```diff
@@ -115,6 +115,35 @@ <h2>A project for first-time contributions</h2>
...
+   <div class="card">
+    <p class="name">Meshack Cheberi</p>
+    <p class="contact">
+      <i class="fa-brands fa-linkedin"></i>
+      <a href="https://www.linkedin.com/in/meshack-cheberi-530a2a220/" target="_blank">LinkedIn</a>
```
Comment (TenTraicion) [inline index.html:123]:
```suggestion
      <a href="https://www.linkedin.com/in/meshack-cheberi-530a2a220/" target="_blank">@meshack-cheberi</a>
```

File: index.html [hunk 3]
```diff
@@ -115,6 +115,35 @@ <h2>A project for first-time contributions</h2>
...
+      <i class="fa-brands fa-github"></i>
+      <a href="https://github.com/Cheberi" target="_blank">Github</a>
```
Comment (TenTraicion) [inline index.html:125]:
```suggestion
      <a href="https://github.com/Cheberi" target="_blank">@Cheberi</a>
```

File: index.html [hunk 4]
```diff
@@ -115,6 +115,35 @@ <h2>A project for first-time contributions</h2>
...
+        <li>
+          <a href="https://www.codecademy.com/" target="_blank" title="Second Resource">Codecademy</a>
+        </li>
+        <li>
+          <a href="https://www.udacity.com/" target="_blank" title="Third resource">Udacity</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <!-- ________ *END TEMPLATE* Contributor card END ________  -->
```
Comment (TenTraicion) [inline index.html:145]:
```suggestion
  <!-- ________ *Meshack Cheberi* Contributor card END ________  -->
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@Cheberi, Undo the CSS file and do the mentioned changes in your card. Thanks!

**Comment by CandidDeer** [comment]
@Cheberi Please note that we will be closing this PR 7 days after opening if the requested changes haven't been made by then. Please feel free to reach out if you have any questions. Thx.

**Comment by TenTraicion** [comment]
@Cheberi, Congratulation on your PR Merge, and Thanks for your Contribution. You can check your card in our [Live Website](https://syknapse.github.io/Contribute-To-This-Project/)! It may take 2-3 minutes to make your card visible in our website. 

Welcome to Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**